x-pack/plugins/aiops/public/components/document_count_content/document_count_chart/document_count_chart.tsx

```diff
@@ -37,14 +39,15 @@ export interface DocumentCountChartPoint {
 }
 
 interface DocumentCountChartProps {
-  brushSelectionUpdateHandler: (d: WindowParameters) => void;
+  brushSelectionUpdateHandler: (d: WindowParameters, force: boolean) => void;
   width?: number;
   chartPoints: DocumentCountChartPoint[];
   chartPointsSplit?: DocumentCountChartPoint[];
   timeRangeEarliest: number;
   timeRangeLatest: number;
   interval: number;
   changePoint?: ChangePoint;
+  isBrushedCleared: boolean;
```

Comment:
`isBrushCleared` might be a simpler name for this prop

Reply:
Fixed in 704d2e2.